packages/block-library/src/comments-pagination/block.json

```diff
@@ -22,7 +22,11 @@
 		"html": false,
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
Most of the children of this block are rendered as links (next, previous, page numbers). Would it be worth including `link` as one of the defaults?